cfgov/jinja2/v1/_includes/organisms/sidebar-contact-info.html

```diff
@@ -50,7 +50,7 @@ <h2 class="header-slug">
             <h3>{{ value.header }}</h3>
         {% endif %}
 
-        {{ parse_links(value.contact.body) |safe }}
+        {{ parse_links(value.contact.body) | safe | richtext }}
     </div>
 
```

Comment:
@richaagarwal  can you verify this https://github.com/cfpb/cfgov-refresh/pull/2131/files#diff-cfbd2caf2990c2080acb5d6bf65684f2R97 doesn't already resolve this issue?